server.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const minimist = require('minimist')
+const Fastify = require('fastify')
+
+function build (opts, callback) {
+  const fastify = Fastify({
+    level: opts['log-level'] || (process.env.NODE_ENV === 'production' ? 'error' : 'info')
+  })
+
+  fastify.get('/status', (req, reply) => {
+    reply.send({ status: 'ok' })
+  })
+
+  fastify.ready(err => {
+    callback(err, fastify, opts)
+  })
```

Comment:
there is no need to call the callback within `ready`, avvio serializes everything.